tests/romtools/test_trial_space.py

```diff
@@ -243,13 +227,12 @@ def test_trial_space_from_scaled_pod():
 
     # test with a shift, splitting, and orthogonalization
     snapshots = np.random.normal(size=(3, 8, 6))
-    snapshot_data = PythonSnapshotData(copy.deepcopy(snapshots))
     my_scaler = utils.VariableScaler('max_abs')
     my_shifter = utils.AverageShifter()
     my_splitter = utils.BlockSplitter([[0], [1, 2]], 3)
     weighting = np.abs(np.random.normal(size=24))
     my_orthogonalizer = utils.EuclideanVectorWeightedL2Orthogonalizer(weighting)
-    my_trial_space = rt.TrialSpaceFromScaledPOD(snapshot_data,
+    my_trial_space = rt.TrialSpaceFromScaledPOD(snapshots,
```

Comment:
why not deepcopy here?

Reply:
Good question. The deep copy is there for the tests so we can compare to stuff derived from the original snapshot matrix. Scaling messes up the original snapshot matrix since the operations are in place. All of the tests but the first one are on the shifted data, which is a copy, so I didn't need the second deep copy.